examples/tinygo-key-value/main.go

```diff
@@ -5,18 +5,15 @@ import (
 	"net/http"
 
 	spin_http "github.com/fermyon/spin/sdk/go/http"
-	"github.com/fermyon/spin/sdk/go/key_value"
+	"github.com/fermyon/spin/sdk/go/kv"
 )
 
 func init() {
 	// handler for the http trigger
 	spin_http.Handle(func(w http.ResponseWriter, r *http.Request) {
-		store, err := key_value.Open("default")
-		if err != nil {
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-			return
-		}
-		defer key_value.Close(store)
+		store := kv.NewStore("default")
+		store.Open()
```

Comment:
Is this a normal pattern in Go to create an object that isn't valid and requires a method call (in this case `Open`) to become valid? It would seem nicer to me to have `NewStore` do the call to open.

Reply:
:+1: `OpenStore`